app/templates/views/organisations/organisation/settings/index.html

```diff
@@ -100,7 +100,28 @@ <h1 class="heading-medium">Settings</h1>
       {% endcall %}
       {% call row() %}
         {{ text_field('Email branding options') }}
-        {{ optional_text_field(current_org.email_branding_pool_names, default="None") }}
+        {% set email_branding_html %}
+          <div {% if other_email_branding_options %}class="govuk-!-margin-bottom-5"{% endif %}>
```

Comment:
Slightly less spacing helps all the items feel part of the same list:
```suggestion
          <div {% if other_email_branding_options %}class="govuk-!-margin-bottom-3"{% endif %}>
```
Before | After
---|---
<img width="567" alt="image" src="https://user-images.githubusercontent.com/355079/195340844-68fe28b5-428d-4ee8-9d58-0635d9c063a0.png"> | <img width="585" alt="image" src="https://user-images.githubusercontent.com/355079/195340782-fbfd57b8-6c32-43d3-a51b-d2536dcc5f35.png">

Reply:
👍